protos/bottle/fulfillment/v1/tax_line.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+message TaxLine {
```

Comment:
Is this going to be used anywhere outside of an Order? If not, I think this is best as a nested resource under Order.

Reply:
It's going to be used for the grpc service. You give it an order and it gives you a response with a list of these.

Reply:
I think it might be better to return the Order populated with these values. In most (all?) of our messages we deal with the top level message in that space typically. Furthermore, these lack any kind of association to the order which I think will be problematic down the road.